streampark-common/src/main/java/org/apache/streampark/common/util/Handler.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.streampark.common.util;
+
+/** A handler with declared exception thrown. */
+public interface Handler<T, R> {
```

Comment:
We don't need the new Fun interface, Just use java.util.function.Function

Reply:
Hi, @wolfboys Thx for the comment.
However, If we use the` java.util.function.Function` without `Exception thrown declaration`, The `Utils.usingAdaptive` will be a lot of `try .... catch ...` clauses, which causes statements noisy.

Of course, I'd like to modify it based on your comments if we could ignored the negative points mentioned above.